access/pagerduty/fake_pagerduty_test.go

```diff
@@ -8,60 +8,74 @@ import (
 	"net/http/httptest"
 	"runtime/debug"
 	"sync"
-	"time"
+	"sync/atomic"
 
-	pd "github.com/PagerDuty/go-pagerduty"
 	"github.com/gravitational/trace"
 	"github.com/julienschmidt/httprouter"
 
 	log "github.com/sirupsen/logrus"
 )
 
 type FakePagerduty struct {
-	srv              *httptest.Server
-	extensions       sync.Map
-	newExtensions    chan pd.Extension
-	incidents        sync.Map
-	newIncidents     chan pd.Incident
-	incidentUpdates  chan pd.Incident
-	newIncidentNotes chan pd.IncidentNote
+	srv *httptest.Server
+
+	objects sync.Map
+	// Extensions
+	extensionIDCounter uint64
```

Comment:
are these counters used anywhere?

Reply:
Yes, for instance look at `StoreExtension`.